src/index.html

```diff
@@ -4,9 +4,444 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="Moyo Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link ">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+              >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <article class="card" data-qa="card">
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="imac-img"
```

Comment:
```suggestion
          alt=" APPLE A1419 iMac 27" Retina"
```
In checklist you can find the point how will be better to provide alt attribute

Reply:
not fixed. 
![2024-01-05_16h02_02](https://github.com/mate-academy/layout_catalog/assets/61904995/c423d265-7dfe-4d6e-a9ab-2f3a383b7f65)
![2024-01-05_16h02_41](https://github.com/mate-academy/layout_catalog/assets/61904995/cf8b2686-f6a6-4b1d-b854-df79c99994fc)